### PR TITLE
consuming_recipes: fix dead link

### PR DESCRIPTION
**Pull request by ericLemanissier**
---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jwillikers — APPROVED** [review]

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
Umm that's not right, I need to check this on my end

File: docs/consuming_recipes.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Here are a few choices:
 - [Cache recipes in your own ArtifactoryCE](https://docs.conan.io/en/latest/uploading_packages/using_artifactory.html) - recommended for production environments
 
 Using your own ArtifactoryCE instance is easy. You can [deploy it on-premise](https://conan.io/downloads.html) or use a
-[cloud provided solution](https://jfrog.com/community/start-free) for **free**. Your project should
+[cloud provided solution](https://jfrog.com/start-free/) for **free**. Your project should
```
Comment (prince-chrismc) [inline docs/consuming_recipes.md:51]:
```suggestion
[cloud provided solution](https://jfrog.com/community/start-free/) for **free**. Your project should
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
This should be working 

image

**Comment by ericLemanissier** [comment]
The link was d'eau this morning, but came back

**Comment by prince-chrismc** [comment]
thanks for letting us know ❤️

**Pull request closed without merging**